sunbeam-python/sunbeam/commands/prepare_node.py

```diff
@@ -111,19 +111,77 @@
 fi
 """
 
+BOOTSTRAP_TEMPLATE = f"""
+# Install the lxd snap
+sudo snap install lxd --channel {LXD_CHANNEL}
+USER=$(whoami)
+# Ensure current user is part of the LXD group
+sudo usermod --append --groups lxd $USER
+
+# Try to determine if LXD is already bootstrapped
+if [ -z "$(sudo --user $USER lxc storage list --format csv)" ];
+then
+    lxdbr=$(sudo --user $USER lxc network list --format csv | grep lxdbr0)
+    if [ -z "$lxdbr" ]; then
+        echo 'Sunbeam requires the LXD bridge to be called anything except lxdbr0'
+        exit 1
+    fi
+
+    echo 'Bootstrapping LXD'
+    cat <<EOF | sudo --user $USER lxd init --preseed
+networks:
+- config:
+    ipv4.address: auto
+    ipv6.address: none
+  name: br0
```

Comment:
Everything looks good except this one minor thing.

What if the machine already has br0 as part of the networking and LXD trying to bootstrap with br0?
Use something like lxdsunbeam0 or lxdsunbeambr0 as bridge name.